bitbucket-pipelines/pulumi-dotnet.yml

```diff
@@ -11,11 +11,12 @@ definitions:
 
 pipelines:
   pull-requests:
-    - step:
-        name: Preview update to infrastructure
-        script:
-          - *vars
-          - pulumi preview --cwd ${PULUMI_WORKING_DIRECTORY} -s ${PULUMI_STACK_NAME}
+    '**':
+      - step:
+          name: Preview update to infrastructure
+          script:
+            - *deps
```

Comment:
Shouldn't this (and in the other lang templates) be `*vars`? The alternative would be to name the template ref defined on line 8 as `&deps` instead of `&vars`.

Reply:
Ping @stack72 @ganeshramcandoit. I just realized I left this comment back in February and missed following-up on it. If you've reviewed my question and still think the template is good, I'll go ahead and resolve this comment.